pyproject.toml

```diff
@@ -115,7 +114,7 @@ mdformat
     SECURITY.md
     tests/README.md
     codegen/README.md
-"""
+"""  # TODO: This is required because the exclude config only works on Python 3.13+
```

Comment:
I just raised https://github.com/hukkin/mdformat/issues/489. It could help to link to that here:

```suggestion
"""
# TODO: Remove once `--exclude` can be used on `python<3.13`, see
# https://github.com/hukkin/mdformat/issues/489
```